includes/woopay/class-woopay-session.php

```diff
@@ -453,6 +453,7 @@ public static function ajax_init_woopay() {
 
 			$woopay_adapted_extensions  = new WooPay_Adapted_Extensions();
 			$body['adapted_extensions'] = $woopay_adapted_extensions->get_adapted_extensions_data( $email );
+			$body['extension_data']     = $woopay_adapted_extensions->get_extension_data();
```

Comment:
Looks good. Thanks Alefe.
Do we need to set this and `adapted_extensions` here or do you think we can pass it in the `store_data` in https://github.com/Automattic/woocommerce-payments/blob/c76a59c993fb0bfe836281d819bef614d86194b7/includes/woopay/class-woopay-session.php#L389

Reply:
We could pass it on store data, but this would not be backwards compatible as a version with `adapted_extensions` was already released, the WooPay side would need to have a condition for both cases until we remove support for older versions, what do you think?

Reply:
ah true. I like the idea of consolidating data passing to WooPay in a single place. If you agree, we can perhaps move `extension_data` data into `store_data` in this PR and migrate `adapted_extensions` in a separate PR.

Reply:
I agree with Malith that the `store_data` seems the right place to be.